k8s/production/ingress-nginx/oauth2-proxy.yml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
```

Comment:
Should we move this `Deployment` and `Service` to `k8s/production/prometheus/ingress.yaml`? Or maybe just move this file itself to `k8s/production/prometheus/`? Since this is oauth2-proxy deployment is specific to prometheus.spack.io, and presumably we'll want to use this for alertmanager.spack.io in the future as well?

Reply:
Done.